requirements.txt

```diff
@@ -1,4 +1,7 @@
 setuptools~=49.6.0
 numpy~=1.20.3
 mdanalysis~=2.0.0.dev0
-pytest~=6.2.4
\ No newline at end of file
+pytest~=6.2.4
+matplotlib~=3.4.2
+nglview~=3.0.3
+scipy~=1.7.0rc1
```

Comment:
Is this necessary? Ideally we want to match the dependency range that someone using the same MDA release would use.

Reply:
It's not necessary, should I remove the specific versioning altogether or specify versions to match the MDA release?

Reply:
@IAlibay has the final say here, but I would lean towards matching the MDA 2.0.0-dev requirements.

Reply:
I matched the MDA requirements specified on the MDA Github. Let me know how the new version looks.

Reply:
LGTM but as before @IAlibay  is the expert on this area of MDA.